docs/diagrams/konnector-workflow.mmdc

```diff
@@ -17,6 +19,7 @@ sequenceDiagram
     K ->> S: login
     K -->> C: [log] login successful
     C -->> H: [ws] login successful
+    K -->> C: update aggregator account
     K ->> S: fetch data
     K ->> C: save documents and files
     C -->>H: [ws] success / error
```

Comment:
Can you also update the png in the repository?

Reply:
Oops, sorry.